src/nxdoc/nuxeo-add-ons/nuxeo-keycloak.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Nuxeo Keycloak
+description: Nuxeo Keycloak allows you to integrate Keycloak, an Open Source Identity and Access Management solution.
+---
+
```

Comment:
Maybe add a shot paragraph explaining the purpose of the addon, like for the other ones (e.g. Kerberos).